apps/lockfile-explorer-web/src/containers/SelectedEntryPreview/styles.scss

```diff
@@ -1,15 +1,10 @@
-@import 'office-ui-fabric-core/dist/sass/References';
```

Comment:
On a smaller responsive window, the `SelectedEntryPreview` can get squished like this:
 
![image](https://user-images.githubusercontent.com/4673363/213329001-b15f2afe-316a-44ca-83a6-a2e2f2c88695.png)

...and also the `renderDependencyMetadata()` pane can get squished:

![image](https://user-images.githubusercontent.com/4673363/213328883-2ffac019-be35-4df4-a0da-690b77dabce8.png)

Here's a couple ways we might address that:

- Since the `SelectedEntryPreview` and `renderDependencyMetadata()` have a fixed amount of content (unlike the other panes), we could force a fixed height to them, maybe just 100% of their content height
- Maybe could also use a responsive breakpoint, so that if the browser window height is too small, then the middle column scrolls (rather than stretching to fill the window height)

Sometimes these CSS layout behaviors can be tricky to implement, so don't spend too much time on it.  But maybe there's some easy fixes.

Reply:
fixed by removing top and bottom padding as well as adding a min height